### PR TITLE
[hi] Localize cloud-native-security.md to Hindi

### LINKED ISSUE #686: [Update term] Cloud_native_security
Updating the term Cloud_native_security for Hindi.

The cloud native security term is not updated :
https://github.com/cncf/glossary/tree/main/content/hi

### PR DESCRIPTION
**Pull request by SaxenaAnushka102**
Signed-off-by: ANUSHKA SAXENA <anushkasaxena102@gmail.com>

### Describe your changes
Created a file `cloud-native-security.md` to localize the page: https://glossary.cncf.io/cloud-native-security/ into Hindi 

### Related issue number 
Resolves #686 

### Checklist before opening this PR 
- [x] This PR does not contain plagiarism
- [x] I have signed off on all commits

### CODE REVIEW

File: content/hi/cloud-native-security.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["सुरक्षा"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव अनुप्रयोगों में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर उत्पादन तक संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है । क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [DevSecOps](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
```
Comment (sayantani11) [inline content/hi/cloud-native-security.md:10]: 1. replace उत्पादन with निर्माण
2.  यह सुनिश्चित करता है कि विकास से लेकर उत्पादन तक, सुरक्षा संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे।

Comment (SaxenaAnushka102) [reply content/hi/cloud-native-security.md]: Thanks for reviewing the PR, @sayantani11. I've made the changes in line 10, please have a look!

File: content/hi/cloud-native-security.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+tags:  ["सुरक्षा"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव अनुप्रयोगों में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर निर्माण तक संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है। क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [DevSecOps](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
+
+## समस्या जिस का यह समाधान करता है
+
+पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब मान्य नहीं हैं। [क्लाउड नेटिव ऐप](/cloud-native-apps/) (अनुप्रयोग) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और बुनियादी ढांचे में तेज़ बदलाव के अधीन होते हैं। कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, मेज़बान-आधारित भेद्यता अवलोकन और अंतिम मिनट की सुरक्षा समीक्षा क्लाउड नेटिव अनुप्रयोगों के साथ [स्केल](/scalability/) नहीं की जा सकती हैं।
```
Comment (sayantani11) [inline content/hi/cloud-native-security.md:14]: 1. मान्य to वैध
2.  why did you use () in अनुप्रयोग
3. @Garima-Negi will it be better to just write the word 'host-based' as it is in hindi or should we localise it?

File: content/hi/cloud-native-security.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+## यह कैसे मदद करता है
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है जो पारंपरिक सुरक्षा प्रतिरूप से स्थानांतरित करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है। हस्तचालित परीक्षण और जांच को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया गया है। त्वरित कोड रिलीज़ पाइपलाइन के उन उपकरणों के साथ एकीकृत हैं जो संकलित होने से पहले भेद्यता के लिए कोड की समीक्षा करते हैं। ओपन सोर्स लाइब्रेरी विश्वसनीय स्रोतों से ली जाती हैं और अरक्षितता के लिए उनकी निगरानी भी की जाती है। बदलाव को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा प्रतिरूप अक्सर अद्यतन किए गए असुरक्षित घटकों द्वारा इसे ग्रहण करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदला जा रहा है।
```
Comment (sayantani11) [inline content/hi/cloud-native-security.md:18]: rephrase this part "करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है" cause it seems out of place. something like -
"...पारंपरिक सुरक्षा प्रतिरूप से स्थानांतरित करती है वहाँ जहां अनुप्रयोगों की सुरक्षा विमोचन (रिलीज़) चक्र के हर कदम में सुरक्षा शामिल होती है।"

**Review by sayantani11 — CHANGES_REQUESTED** [review]

File: content/hi/cloud-native-security.md (hunk 2 above)
Comment (SaxenaAnushka102) [reply content/hi/cloud-native-security.md]: Hi @sayantani11, thanks for the feedback. Here's the progress:
1. Done👍
2.  'अनुप्रयोग' is the Hindi localization for app. I've put in within brackets with the purpose of providing further context to the reader about the technical term "cloud-native apps". I thought to provide additional information to clarify the meaning of the term. What are your thoughts on this?
3. I too agree with you on this, 'मेज़बान-आधारित' would better be replaced by 'host-based' as it is more of a technical term and may be more commonly known in its English form.💯

File: content/hi/cloud-native-security.md (hunk 3 above)
Comment (SaxenaAnushka102) [reply content/hi/cloud-native-security.md:18]: How about replacing the whole sentence with "क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीका पेश करती है। यह पारंपरिक सुरक्षा प्रतिरूप को स्थानांतरित करके अनुप्रयोगों की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है।" ?

File: content/hi/cloud-native-security.md (hunk 2 above)
Comment (Garima-Negi) [reply content/hi/cloud-native-security.md]: I'd go with host-based instead of मेज़बान-आधारित. 
We can also eliminate अनुप्रयोग IMO.

Comment (SaxenaAnushka102) [reply content/hi/cloud-native-security.md]: Hi @Garima-Negi! Thanks for the suggestions, I've incorporated them all in the latest commit.

**Review by Garima-Negi — APPROVED** [review]

File: content/hi/cloud-native-security.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+tags:  ["Security"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव ऐपस् में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर निर्माण तक संपूर्ण ऐप के जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है। क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [(DevSecOps)](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
+
+## समस्या
+
+पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब वैध नहीं हैं। [क्लाउड नेटिव ऐपस्](/cloud-native-apps/) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और बुनियादी ढांचे में तेज़ बदलाव के अधीन होते हैं। कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, होस्ट पर आधारित भेद्यता अवलोकन और अंतिम मिनट की सुरक्षा समीक्षा क्लाउड नेटिव ऐपस् के साथ [स्केल](/scalability/) नहीं की जा सकती हैं।
```
Comment (aj11anuj) [inline content/hi/cloud-native-security.md:14]:
```suggestion
पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब वैध नहीं हैं। 
[क्लाउड नेटिव ऐपस्](/cloud-native-apps/) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, 
अक्सर विक्रेता-नियंत्रित इंफ्रास्ट्रक्चर में चलते हैं, और इस तेज़ बदलाव के अधीन होते हैं। 
कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, होस्ट पर आधारित भेद्यता अवलोकन और अंतिम मिनट की 
सुरक्षा समीक्षा क्लाउड नेटिव ऐपस् के साथ [स्केल](/scalability/) नहीं की जा सकती हैं। 
```

File: content/hi/cloud-native-security.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+पारंपरिक सुरक्षा प्रतिरूप कई मान्यताओं के साथ बनाए गए थे जो अब वैध नहीं हैं। [क्लाउड नेटिव ऐपस्](/cloud-native-apps/) बार-बार बदलते हैं, बड़ी संख्या में ओपन सोर्स संसाधनों और लाइब्रेरी का उपयोग करते हैं, अक्सर विक्रेता-नियंत्रित बुनियादी ढांचे में चलते हैं, और बुनियादी ढांचे में तेज़ बदलाव के अधीन होते हैं। कोड समीक्षा, लंबे गुणवत्ता आश्वासन चक्र, होस्ट पर आधारित भेद्यता अवलोकन और अंतिम मिनट की सुरक्षा समीक्षा क्लाउड नेटिव ऐपस् के साथ [स्केल](/scalability/) नहीं की जा सकती हैं। 
+
+## समाधान
+
+क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीक़ा पेश करती है। यह पारंपरिक सुरक्षा प्रतिरूप को स्थानांतरित करके ऐपस् की सुरक्षा करती है जहां विमोचन (रिलीज़) चक्र के हर चरण में सुरक्षा शामिल होती है। हस्तचालित परीक्षण और जांच को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया गया है। त्वरित कोड रिलीज़ पाइपलाइन के उन उपकरणों के साथ एकीकृत हैं जो संकलित होने से पहले भेद्यता के लिए कोड की समीक्षा करते हैं। ओपन सोर्स लाइब्रेरी विश्वसनीय स्रोतों से ली जाती हैं और अरक्षितता के लिए उनकी निगरानी भी की जाती है। बदलाव को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा प्रतिरूप अक्सर अद्यतन किए गए असुरक्षित घटकों द्वारा इसे ग्रहण करता है या यह सुनिश्चित करता है कि बुनियादी ढांचे को नियमित रूप से बदला जा रहा है।
```
Comment (aj11anuj) [inline content/hi/cloud-native-security.md:18]:
```suggestion
क्लाउड नेटिव सुरक्षा काम करने का एक नया तरीक़ा पेश करती है। यह पारंपरिक सुरक्षा प्रतिरूप को स्थानांतरित करके 
ऐपस् की सुरक्षा करती है, जहां रिलीज़ चक्र के हर चरण में सुरक्षा शामिल होती है। 
हस्तचालित परीक्षण और जांच को बड़े पैमाने पर स्वचालित स्कैन से बदल दिया गया है। 
त्वरित कोड रिलीज़ पाइपलाइन के उन उपकरणों के साथ एकीकृत हैं जो संकलित होने से पहले भेद्यता के लिए कोड की समीक्षा करते हैं। 
ओपन सोर्स लाइब्रेरी विश्वसनीय स्रोतों से ली जाती हैं और अरक्षितता के लिए उनकी निगरानी भी की जाती है। 
बदलाव को धीमा करने के बजाय एक क्लाउड नेटिव सुरक्षा प्रतिरूप अक्सर अद्यतन किए गए असुरक्षित घटकों द्वारा इसे ग्रहण करती है, और
यह सुनिश्चित करती है कि इंफ्रास्ट्रक्चर को नियमित रूप से बदला जा रहा है।
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Apart from these suggestions, there are many heavy hindi words used in 2nd and 3rd paragraph. Consider replacing them with simple words.

**Review by aj11anuj — APPROVED** [review]

**Review by kumarankit999 — COMMENTED** [review]
lgtm

**Pull request merged**